test/server/integration/TimeLimitExpired.spec.js

```diff
@@ -0,0 +1,96 @@
+                beforeEach(function() {
+                    expect(this.game.finishedAt).toBeUndefined();
+                    expect(this.game.winner).toBeUndefined();
+                    this.completeSetup();
+                    this.selectFirstPlayer(this.player1);
+                    this.completeMarshalPhase();
+                    this.game.addPower(this.player1Object, 10);
+                    this.game.addPower(this.player2Object, 10);
+                    this.player1.clickPrompt('Done');
+                });
+
+                it('based on which player is closer to 15 power', function() {
+                    this.game.addPower(this.player1Object, 1);
+                    this.player2.clickPrompt('Done');
+                    this.game.determineWinnerAfterTimeLimitExpired();
+                    expect(this.game.finishedAt).toBeDefined();
+                    expect(this.game.winner).toBe(this.player1Object);
+                });
+
+                it('based on which player has more cards left in his/her deck', function() {
+                    this.player1Object.drawCardsToHand(1);
+                    this.player2.clickPrompt('Done');
+                    this.game.determineWinnerAfterTimeLimitExpired();
+                    expect(this.game.finishedAt).toBeDefined();
+                    expect(this.game.winner).toBe(this.player2Object);
+                });
+
+                it('based on which player won dominance in the last round', function() {
+                    this.game.addGold(this.player2Object, 1);
+                    this.player2.clickPrompt('Done');
+                    expect(this.game.winnerOfDominanceInLastRound).toBe(this.player2Object);
+                    this.game.determineWinnerAfterTimeLimitExpired();
+                    expect(this.game.finishedAt).toBeDefined();
+                    expect(this.game.winner).toBe(this.player2Object);
+                });
+
+                it('based on which player has the fewest characters in his/her dead pile', function() {
+                    this.deadCard = this.player1.findCardByName('Advisor to the Crown', 'hand');
+                    this.player1Object.moveCard(this.deadCard, 'dead pile');
+                    this.player2.clickPrompt('Done');
+                    this.game.determineWinnerAfterTimeLimitExpired();
+                    expect(this.game.finishedAt).toBeDefined();
+                    expect(this.game.winner).toBe(this.player2Object);
+                });
+    
+                it('based on which player was the first player in the last round', function() {
+                    this.game.determineWinnerAfterTimeLimitExpired();
+                    expect(this.game.finishedAt).toBeDefined();
+                    expect(this.game.winner).toBe(this.player1Object);
+                });
+    
+                it('when one player cannot win the game', function() {
+                    this.tycho = this.player1.findCardByName('Tycho Nestoris', 'hand');

```

Comment:
Given how many non-Tycho cards are in this test deck, he's not guaranteed to be in hand here. If you just omit the second parameter, it should search through all areas for the player to find a copy.